frontend/web/styles/project/_modals.scss

```diff
@@ -211,7 +211,28 @@ $side-width: 750px;
     max-height: 350px;
   }
 }
-
+@include media-breakpoint-down(lg) {
+  .inline-modal__list {
+    max-height: none;
+  }
+  .inline-modal, .inline-modal.table-filter {
+    position: fixed;
+    overflow-y: auto;
+    top: 0;
+    left: 0;
+    right: 0;
+    bottom: 0;
+    z-index: 100000000000;
+    margin: 0;
+    width: 100vw;
+    max-width: 100vw;
+    height: 100vh;
+    max-height: 100vh;
+  }
+  .table-filter-list {
+    max-height: none !important;
+  }
+}
 .modal-back-btn {
```

Comment:
<img width="303" height="172" alt="image" src="https://github.com/user-attachments/assets/3314e799-8edb-47a5-9832-df26d68b71a7" />


There is just also those 2 buttons which could be a low hanging fruits to tackle if ever :D